internal/commands/scan.go

```diff
@@ -826,6 +828,24 @@ func setupScanTypeProjectAndConfig(
 	if findProjectErr != nil {
 		return findProjectErr
 	}
+
+	importFileType, err := cmd.Flags().GetString(commonParams.ImportFileType)
+	if err != nil {
+		return err
+	}
+	importFilePath, err := cmd.Flags().GetString(commonParams.ImportFilePath)
+	if err != nil {
+		return err
+	}
+	if importFileType != "" && importFilePath != "" {
+		_, importFileError := importFile(projectID, importFileType, importFilePath)
+		if importFileError != nil {
+			logger.Printf(importFileError.Error())
```

Comment:
write your error here maybe / make sure we will not miss the error message we want to shoe the customer

Reply:
the error we will show to the customer is mapped inside importFile, which will be implemented by Tamar